cheroot/connections.py

```diff
@@ -203,11 +204,23 @@ def run(self, expiration_interval):
             self._serving = False
 
     def _run(self, expiration_interval):
+        """Run connection handler loop until stop was requested.
+
+        Use expiration_interval as select() timeout
+        to assure expired connections are closed in time.
+
+        On Windows cap the timeout to 0.05 seconds
+        as select() does not return when a socket is ready.
+        """
         last_expiration_check = time.time()
+        if os.name == 'nt':
+            select_timeout = min(expiration_interval, 0.05)
+        else:
+            select_timeout = expiration_interval
```

Comment:
There's a constant in the `_compat` module, reuse it: https://github.com/cherrypy/cheroot/blob/master/cheroot/_compat.py
```suggestion
        select_timeout = min(expiration_interval, 0.05 if IS_WINDOWS else expiration_interval)
```

Reply:
Considering, we need `0.01` for *NIX, it'd be
There's a constant in the `_compat` module, reuse it: https://github.com/cherrypy/cheroot/blob/master/cheroot/_compat.py
```suggestion
        select_timeout = min(expiration_interval, 0.05 if IS_WINDOWS else 0.01)
```

Reply:
But since these are magic numbers, we need to move them into a module-level constant with a proper name to give these floats a human-readable label. It'd look better like this:
```python
# module-top
from ._compat import IS_WINDOWS
...
_DEFAULT_SELECT_TIMEOUT = 0.05 if IS_WINDOWS else 0.01
```
```suggestion
        select_timeout = min(expiration_interval, _DEFAULT_SELECT_TIMEOUT)
```